ui/imports/shared/views/AssetsDetailView.qml

```diff
@@ -43,8 +43,8 @@ Item {
         asset.name: token && token.symbol ? Style.png("tokens/%1".arg(token.symbol)) : ""
         asset.isImage: true
         primaryText: token ? token.name : ""
-        secondaryText: token ? qsTr("%1 %2").arg(Utils.toLocaleString(token.totalBalance, RootStore.locale, {"currency": true})).arg(token.symbol) : ""
-        tertiaryText: token ? "%1 %2".arg(Utils.toLocaleString(token.totalCurrencyBalance.toFixed(2), RootStore.locale, {"currency": true})).arg(RootStore.currencyStore.currentCurrency.toUpperCase()) : ""
+        secondaryText: token ? `${token.enabledNetworkBalance} ${token.symbol}` : ""
```

Comment:
using qstr caused the bug, there is no need to use translation or formatting as they should always be displayed as they are

Reply:
I doubt it... I'd bet on the other function there 😊